layout/composite.c

```diff
@@ -65,7 +65,7 @@ M0_INTERNAL void m0_composite_build(struct m0_layout_domain *dom,
 
 #if 0 /* unused code */
 /** Implementation of lo_fini for COMPOSITE layout type. */
-static void composite_fini(struct m0_ref *ref)
+M0_UNUSED  static void composite_fini(struct m0_ref *ref)
```

Comment:
How are these change related to ARM porting? If they are not, please move them all from here to a separate PR. They are distracting and take everyone's reviewer time. It's annoying.

Reply:
As per  huang hua suggestion ,the set of unused function are maked with M0_UNUSED from making the compiler warn.

Reply:
These unused functions will generate error on ARM platform?
So, I suggested to mark them as M0_UNUSED instead of remove them.
Because maybe in the future, they will be used again.

Reply:
Already marked all of them with M0_UNUSED macro for smooth compilation